README.md

```diff
@@ -0,0 +1,85 @@
+
+1. Who's in the team.
+2. Which squad each individual belongs to and availability.
+3. OSS maintained projects.
+4. Release process.
+5. Technical documents.
+6. New hires checklist.
+7. Interview process.
+8. Code guidelines, standards and etiquette.
+
+
+## 1. Who's in the team
+
+| Name                    | Contact                                                       |  
+|-------------------------|---------------------------------------------------------------|
+| Ana Catarina Figueiredo | [@AnnKatFig](https://twitter.com/AnnKatFig)                   |
+| Anders Ha               | [@_andersha](https://twitter.com/_andersha)                   |
+| Ben Henshall            |                                                               |
+| Danilo Aliberti         |                                                               |
+| David Rodrigues         | [@dmcrodrigues](https://twitter.com/dmcrodrigues)             |
+| Diego Petrucci          | [@diegopetrucci](https://twitter.com/diegopetrucci)           |
+| Ilya Puchka             | [@ilyapuchka](https://twitter.com/ilyapuchka)                 |
+| Jason Dobo              |                                                               |
+| Martin Nygren           |                                                               |
+| Michael Brown           | [@mluisbrown](https://twitter.com/mluisbrown)                 |
+| Rui Peres               | [@peres](https://twitter.com/peres)                           |
+| Sergey Shulga           | [@SergDort](https://twitter.com/SergDort)                     |
+| Viorel Mihalache        | [@viorelMO](https://twitter.com/viorelMO)                     |
+| Wagner Truppel          |                                                               |    
+| Witold Skibniewski      | [@mr_witold](https://twitter.com/mr_witold)                   |
+
+
+## 2. Which squad each individual belongs to and availability
+
+By definition members work on their respective squad, although they are free to work in different squads if the work load justifies it.
+
+
+| Squad Name                    | Members                  | Availability |  
+|-------------------------------|--------------------------| ------------ |
+| SDK                           | David, Viorel, Martin    |    3/4       |
+| Consultation Experience       | Ilya                     |    1/1       |
+| Healthcheck                   | Ben, Wagner              |    2/2       |
+| Core                          | Witold, Jason            |    2/4       |
+| Professional Services         | Danilo                   |    1/1       |
+| GP at Hand                    | Diego                    |    1/1       |
+| Project Avalon                | Sergey                   |    1/1       |
+| Project Minkowski             | Anders                   |    1/2       |
+| Project Bayes                 | Michael                  |    1/1       |
+
+## 3. OSS maitained projects
```

Comment:
```suggestion
## 3. OSS maintained projects
```

Reply:
Actually, it should have an hyphen too: "OSS-maintained"